app/controllers/root_controller.rb

```diff
@@ -31,10 +31,7 @@ def govuk_locales
   end
 
   NON_LAYOUT_TEMPLATES = %w(
-    alpha_label
     barclays_epdq
```

Comment:
This might belong in a separate PR, but this also seems to be no longer used either. I think it was in transaction-wrappers, which have moved to [gds-attic](https://github.com/gds-attic/transaction-wrappers/search?utf8=%E2%9C%93&q=epdq)

Search of github.com/alphagov:
https://github.com/search?q=user%3Aalphagov+barclays_epdq&type=Code&utf8=%E2%9C%93